### DISCUSSION TITLE
is ARC relying on "summerwind" individual?

### DISCUSSION THREAD

**Discussion by dongho-jung** [actions/actions-runner-controller#1697] [Questions] [answered]
in README, there are quotes like "The default "summerwind/actions-runner" images are available at DockerHub" and "apiVersion: actions.summerwind.dev/v1alpha1" ...

I wonder what if he decides to stop maintaining this project? I remember a maintainer who owns an open source zappa having almost 2k stars stopped to maintain the project because he didnt have the time to keep that alive..

**Comment by toast-gear** [comment, marked as answer]
all images also get published to GitHub Packages https://github.com/orgs/actions-runner-controller/packages?repo_name=actions-runner-controller so we don't solely use the summerwind account in DockerHub, we do have an issue raised to move to another DockerHub account but havent got around to it yet.

**Reply by dongho-jung** [reply]
Thanks for your answer and work for this awesome project : )
